### PR TITLE
fix slug bug

### PR DESCRIPTION
**Pull request by sammymwangangi**

### CODE REVIEW

File: src/app/videos/[slug]/page.tsx [hunk 1]
```diff
@@ -155,34 +155,36 @@ const VideoDetails = ({ params }: { params: { slug: string } }) => {
...
-        )
-
-        setVideoDetails({ ...video })
-        setIsLoading(false)
+        );
+        const data = await response.json();
+        console.log('API Response:', data); // Log the entire response
+  
+        const videos = data.data;
+        console.log('Videos:', videos); // Log the videos array
```
Comment (mconyango) [inline src/app/videos/[slug]/page.tsx:163]: @sammymwangangi is it possible to restrict `console.log` to dev and uat environment only but not production? You could use an environment variable and check if env!=='prod' then you do `console.log`. We do not want to expose much details in prod environment for obvious security reasons.

Comment (sammymwangangi) [reply src/app/videos/[slug]/page.tsx:163]: Yes it is possible. That makes sense. I will work on it.

File: src/app/videos/[slug]/page.tsx [hunk 2]
```diff
@@ -155,34 +155,36 @@ const VideoDetails = ({ params }: { params: { slug: string } }) => {
...
+  
+        // Find the video by slug
+        const video = videos.find((v: Video) => v.slug === slug);
+        console.log(video);
+  
+        if (video) {
+          const vimeoVideoId = video.vimeo_link?.split('/').pop();
+          setVideoUrl(
+            `https://player.vimeo.com/video/${vimeoVideoId}?badge=0&autoplay=1&title=0&player_id=0&app_id=58479`,
```
Comment (mconyango) [inline src/app/videos/[slug]/page.tsx:172]: `app_id=58479`, can we define `58479` as constant.

Comment (sammymwangangi) [reply src/app/videos/[slug]/page.tsx:172]: Absolutely.

**Pull request merged**